modules/jaeger-deployment-best-practices.adoc

```diff
@@ -12,4 +12,6 @@ This module included in the following assemblies:
 
 * If you have a multitenant implementation and tenants are separated by namespaces, deploy a Jaeger instance to each tenant namespace.
 
+** Jaeger agent as sidecar is the only supported configuration. Jaeger as daemonset is not supported on for multitenant installations or on OpenShift Dedicated.
```

Comment:
Suggest: as sidecar/as a sidecar. 
Suggest: as daemonset/as a daemonset
Remove extra 'on' from 'not supported on for multitenant'